sdk/Lusid.Sdk.Tests/tutorials/MarketData/Quotes.cs

```diff
@@ -0,0 +1,108 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Lusid.Sdk.Api;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using Lusid.Sdk.Utilities;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.MarketData
+{
+    [TestFixture]
+    public class Quotes
+    {
+        private IQuotesApi _quotesApi;
+
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+            var apiFactory = LusidApiFactoryBuilder.Build("secrets.json");
+            _quotesApi = apiFactory.Api<IQuotesApi>();
+        }
+
+        [Test]
+        public void Add_Quote()
+        {
+            var request = new UpsertQuoteRequest(
+                quoteId: new QuoteId(
+                    provider: "DataScope",
+                    priceSource: "USDRC",
```

Comment:
We should change the default priceSource to be "" as this is the new default.

Reply:
Removed